sdcm/cluster_k8s/__init__.py

```diff
@@ -2339,7 +2339,9 @@ def add_nodes(self,
                   ec2_user_data: str = "",
                   dc_idx: int = 0,
                   rack: int = 0,
-                  enable_auto_bootstrap: bool = False) -> List[BasePodContainer]:
+                  enable_auto_bootstrap: bool = False,
```

Comment:
I think we should add check for K8S methods and raise some exception if it is not `None`.
Why? Because this new parameter is not compatible with theScylla nodes/pods provisioning on K8S platform.

Reply:
added an assert here